docs/app/Examples/elements/Label/Types/LabelExampleAttached.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { Grid, Image, Label, Segment } from 'stardust'
+
+const imageStyles = {
```

Comment:
I don't think we should ever include styles in the examples.  I think it gives the impression that we require inline styles to be used for the example to work.  If we truly _must_ include styles, there are doc site css rules in the `index.ejs` template for this, but let's not put them in the JSX.

In this case, I think there is little enough difference that we can just leave them out:

![http://g.recordit.co/c0Js946Z9x.gif](http://g.recordit.co/c0Js946Z9x.gif)

Alternatively, we could use `<Segment padded>` to give whitespace similar to what the inline image styles did:

![http://g.recordit.co/EZpy8uQuZp.gif](http://g.recordit.co/EZpy8uQuZp.gif)